### PR TITLE
gen: calculate pyver for py3 with -lang cffi

### PR DESCRIPTION
**Pull request by lifei**
Fixes go-python/gopy#145.

### CODE REVIEW

**Comment by corona10** [comment]
Awesome!

File: gen.go [hunk 1]
```diff
@@ -47,6 +47,16 @@ func genPkg(odir string, p *bind.Package, lang string) error {
 		pyvers = 2
 	case "python3", "py3":
 		pyvers = 3
+	case "go", "cffi":
+		pylang, err := getPythonVersion()
```
Comment (sbinet) [inline gen.go]: if `getPythonVersion()` fails for some reason, we shadow the `err` variable from earlier.

you should do:
```go
var pylang string
pylang, err = getPythonVersion()
```

Comment (lifei) [reply gen.go]: thanks. I fix it.

Comment (lifei) [reply gen.go]: done

**Comment by lifei** [comment]
something wrong with travis ci.

**Comment by rcoreilly** [comment]
I'm closing this as it is no longer applicable to the current version.

**Pull request closed without merging**